.github/workflows/java-model-diff.yml

```
@@ -0,0 +1,104 @@
+name: Models as Data
+
+on:
+  workflow_dispatch:
+    inputs:
+      projects:
+        description: "The projects to generate models for"
+        required: true
+        default: '["netty/netty"]'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "java/ql/src/utils/model-generator/**/*.*"
+
+jobs:
+  model-diff:
+    name: Model Difference
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # large
+        # ["google/guava", "spring-projects/spring-framework", "apache/poi"]
+        # others
+        # ["FasterXML/jackson-core", "FasterXML/jackson-databind", "google/gson", "JodaOrg/joda-time"
+        slug: ${{fromJson(github.event.inputs.projects || '["apache/commons-codec", "apache/commons-io", "apache/commons-beanutils", "apache/commons-logging", "apache/commons-fileupload", "apache/commons-lang", "apache/commons-validator", "apache/commons-csv", "apache/dubbo"]' )}}
+    steps:
+      - name: Clone self (github/codeql) for baseline
+        uses: actions/checkout@v2
+        with:
+          path: codeql-baseline
+          ref: ${{ github.base_ref }}
+      - name: Clone self (github/codeql) with new generator
+        uses: actions/checkout@v2
+        with:
+          path: codeql-head
+          ref: ${{ github.ref }}
+      - uses: ./codeql-baseline/.github/actions/fetch-codeql
+      - name: Download database
+        run: |
+          set -x
+          mkdir lib-dbs
+          SHORTNAME=${SLUG//[^a-zA-Z0-9_]/}
+          projectId=`curl -s https://lgtm.com/api/v1.0/projects/g/${matrix.slugs}} | jq .id`
```

After the above change.
```suggestion
          projectId=`curl -s "https://lgtm.com/api/v1.0/projects/g/${SLUG}" | jq .id`
```